src/js/chapterEnd.js

```diff
@@ -0,0 +1,39 @@
+//import renderContent function. 
+
+//quiz will be a node that can be embedded into chapter end page. 
+const createChapterEnd = (chapterNumber, quiz, nextPage, homePage) => {
+    const base = document.createElement("div");
+    base.setAttribute('id', 'chapter-end')
+
+    let quizPassed = false;
+    let toggleQuizStatus = () => quizPassed = true; 
+
+    let text = document.createElement("p");
+    text.innerHTML = `Chapter ${chapterNumber} complete!`
+
+    let nextButton = document.createElement("button");
+    nextButton.innerHTML = `Next Chapter`;
+    //if quiz has not been passed yet, do NOT display next chapter button
+    nextButton.style.display = (quizPassed)? "block" : "none";
```

Comment:
I don't understand how nextButton changes from hidden to visible... Do you want to invoke createChapterEnd again if quiz is passed?

Something else: there is this method `element.classList.toggle('class')`. Could be a good fit here, instead of chaning style directly in JS.